birdbot.py

```diff
@@ -62,6 +62,8 @@
     global commandqueue
     if not message.content.startswith(mprefix):
         return
+    if message.author.voice.voice_channel == None:
```

Comment:
This will stop all commands, you'd need to put this check in the voice specific ones.